src/frontends/tensorflow/src/frontend.cpp

```diff
@@ -227,6 +230,12 @@ ov::frontend::InputModel::Ptr FrontEnd::load_impl(const std::vector<ov::Any>& va
         // this is used for OpenVINO with TensorFlow Integration
         auto graph_iterator = variants[0].as<GraphIterator::Ptr>();
         return std::make_shared<InputModel>(graph_iterator, m_telemetry);
+    } else if (variants[0].is<std::shared_ptr<IGraphIterator>>()) {
+        auto i_graph_iterator = variants[0].as<std::shared_ptr<IGraphIterator>>();
+        FRONT_END_GENERAL_CHECK(i_graph_iterator, "Couldn't cast ov::Any to IGraphIterator");
```

Comment:
Please correct the message: `[TensorFlow Frontend] internal error: could not cast ov::Any to IGraphIterator`
The same comment below

Reply:
Done.